packages/celo-eslint/index.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'plugin:workspaces/recommended',
+  ],
+  plugins: ['import', 'unused-imports', 'jsx-a11y', 'prettier'],
+  rules: {
+    'prettier/prettier': 'warn',
+    'semi': 'off',
+    'import/no-anonymous-default-export': 'warn',
+    'unused-imports/no-unused-imports': 'warn',
+
+    // A11y
+    'jsx-a11y/alt-text': [
+      'warn',
+      {
+        elements: ['img'],
+        img: ['Image'],
+      },
+    ],
+    'jsx-a11y/aria-props': 'warn',
+    'jsx-a11y/aria-proptypes': 'warn',
+    'jsx-a11y/aria-unsupported-elements': 'warn',
+    'jsx-a11y/role-has-required-aria-props': 'warn',
+    'jsx-a11y/role-supports-aria-props': 'warn',
+
+    // Typescript
+    '@typescript-eslint/no-unused-vars': [
+      'warn',
+      {
+        vars: 'all',
+        varsIgnorePattern: '^_',
+        args: 'after-used',
+        argsIgnorePattern: '^_',
+      },
+    ],
+  },
+  parser: '@typescript-eslint/parser',
+  ignorePatterns: ['*.js', 'lib/'],
```

Comment:
If I understand correctly, `lib/` would imply/require that this is the output dir of the compiled code? If so, that might change per project and we should probably not add it here?